src/app/lib/analyticsUtils/sendBeacon/index.test.js

```diff
@@ -61,20 +61,76 @@ describe('sendBeacon', () => {
       },
     };
 
-    it('should call Reverb viewEvent if Reverb config is passed', async () => {
-      const sendBeacon = require('./index').default;
+    const originalProcessEnv = process.env;
 
-      await sendBeacon('https://foobar.com', reverbConfig);
+    afterEach(() => {
+      process.env = originalProcessEnv;
+    });
+
+    describe('LOCAL', () => {
+      beforeEach(() => {
+        process.env.SIMORGH_APP_ENV = 'local';
+      });
```

Comment:
Instead of this, would it be easier to mock the isLive() function, so that there are just 2 scenarios - 1: it is live, 2: it is not live? We've done a similar thing here: https://github.com/bbc/simorgh/blob/f833968fe5dc9cc20dd5cdb12f73b306a2516a13/src/app/components/MediaLoader/utils/buildSettings.test.ts#L758 and https://github.com/bbc/simorgh/blob/f833968fe5dc9cc20dd5cdb12f73b306a2516a13/src/app/components/MediaLoader/utils/buildSettings.test.ts#L771

Reply:
Thanks for the feedback Karina.

I considered mocking `isLive` and instead opted to mock `process.env.SIMORGH_APP_ENV`
to ensure the actual `isLive` implementation  is invoked for the Reverb check.

This is in keeping with other tests that override `process.env.SIMORGH_APP_ENV`,
to check assert environment behaviour.

Reply:
That's fair enough, happy to leave as is.